lib/presentation/engagement/home/pages/home_page.dart

```diff
@@ -126,7 +126,9 @@ class HomePage extends StatelessWidget {
           },
         ),
       ),
-      bottomNavigationBar: const BottomNavBar(),
+      bottomNavigationBar: const BottomNavBar(
+        bottomNavIndex: 0,
```

Comment:
Replace this with the enum index to ensure consistency in case it changes positions in future.

More information [here](https://api.dart.dev/stable/2.16.1/dart-core/Enum/index.html)

```suggestion
        bottomNavIndex: BottomNavIndex.home.index,
```